src/ngAnimate/animateQueue.js

```diff
@@ -579,6 +586,11 @@ var $$AnimateQueueProvider = ['$animateProvider', function($animateProvider) {
       var animateChildren;
       var elementDisabled = disabledElementsLookup.get(getDomNode(element));
 
+      if (elementDisabled === true) {
+        // If animations on the element have been disabled explicitly, no need to traverse the parents
+        return false;
+      }
```

Comment:
I'll remove this, because this is checked already at https://github.com/angular/angular.js/pull/13784/files#diff-c15d87c15661cba4a721501ba4fda5b2R342